tailscale/client_test.go

```diff
@@ -1298,3 +1298,51 @@ func TestClient_RotateWebhookSecret(t *testing.T) {
 	assert.Equal(t, "/api/v2/webhooks/54321/rotate", server.Path)
 	assert.Equal(t, expectedWebhook, actualWebhook)
 }
+
+func TestClient_Contacts(t *testing.T) {
+	t.Parallel()
+
+	client, server := NewTestHarness(t)
+	server.ResponseCode = http.StatusOK
+
+	expectedContacts := &tailscale.Contacts{
+		Account: tailscale.Contact{
+			Email:             "test@example.com",
+			FallbackEmail:     "test2@example.com",
+			NeedsVerification: false,
+		},
+		Support: tailscale.Contact{
+			Email:             "test3@example.com",
+			NeedsVerification: false,
+		},
+		Security: tailscale.Contact{
+			Email:             "test4@example.com",
+			FallbackEmail:     "test5@example.com",
+			NeedsVerification: true,
+		},
+	}
+	server.ResponseBody = expectedContacts
+
+	actualContacts, err := client.Contacts(context.Background())
+	assert.NoError(t, err)
+	assert.Equal(t, http.MethodGet, server.Method)
+	assert.Equal(t, "/api/v2/tailnet/example.com/contacts", server.Path)
+	assert.Equal(t, expectedContacts, actualContacts)
+}
+
+func TestClient_UpdateContact(t *testing.T) {
+	t.Parallel()
+
+	client, server := NewTestHarness(t)
+	server.ResponseCode = http.StatusOK
+	server.ResponseBody = nil
+
+	email := "new@example.com"
+	updateRequest := tailscale.UpdateContactRequest{
+		Email: &email,
+	}
+	err := client.UpdateContact(context.Background(), tailscale.ContactAccount, updateRequest)
+	assert.NoError(t, err)
+	assert.Equal(t, http.MethodPatch, server.Method)
+	assert.Equal(t, "/api/v2/tailnet/example.com/contacts/account", server.Path)
```

Comment:
If you wanted to, you could add the below to verify that the server got the expected body:

```
var receivedRequest tailscale.UpdateContactRequest
err = json.Unmarshal(server.Body.Bytes(), &receivedRequest)
assert.NoError(t, err)
assert.EqualValues(t, updateRequest, receivedRequest)
```